python/ray/data/tests/test_streaming_integration.py

```diff
@@ -103,6 +103,17 @@ def run(self):
     c1.start()
     c0.join()
     c1.join()
+
```

Comment:
We could also add a unit test to test_operators.py for OutputSplitter

Reply:
added